src/tokens.js

```diff
@@ -2086,6 +2166,26 @@ module.exports = [
                     },
                 },
             },
+            {
+                group: 'Body 1',
+                id: 'body__1__weight',
+                platforms: {
+                    ios: {
+                        name: 'text1Weight',
+                        value: 'FontWeight = .normal',
+                    },
+                },
+            },
```

Comment:
Should we do `Text 1` since we're trying to move away from "Body"?

https://thumbprint.design/tokens/ios/#section-font
https://github.com/thumbtack/thumbprint-tokens/issues/4

Reply:
I think we should do the change for all the tokens at once. The `group` needs to be the same for all of the tokens anyway, right?

Reply:
Oh, I didn't notice that the `name` property here uses the word "text". This makes sense!